README.md

```diff
@@ -4,7 +4,7 @@
 
 ![build](https://github.com/TwinProduction/go-away/workflows/build/badge.svg)
 [![Go Report Card](https://goreportcard.com/badge/github.com/TwinProduction/go-away)](https://goreportcard.com/report/github.com/TwinProduction/go-away)
-[![codecov](https://codecov.io/gh/TwinProduction/go-away/branch/master/graph/badge.svg)](https://codecov.io/gh/TwinProduction/go-away)
+[![codecov](https://codecov.io/gh/TwinProduction/go-away/branch/main/graph/badge.svg)](https://codecov.io/gh/TwinProduction/go-away)
```

Comment:
i wasn't sure about this one... would this look for the branch by name or does something else need to change?